test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -118,6 +146,48 @@
            (add-ring-handler proxy-webserver# ~proxy-handler ~endpoint))
          ~@body)))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;; Core Helpers
+
+(deftest json-response-test
+  (testing "json response"
+    (let [source {"key1" "val1", "key2" "val2"}
```

Comment:
Ahh, good catch. That was the one test that I was able to [reuse from existing repos](https://github.com/puppetlabs/puppetserver/blob/master/test/unit/puppetlabs/puppetserver/ringutils_test.clj#L45-L54)!

That reminds me that I haven't used cljfmt against this. After I address any other code concerns I will run that against this PR (from my user profile).